osu.Game/Overlays/RestoreDefaultValueButton.cs

```diff
@@ -92,6 +97,10 @@ protected override void OnHoverLost(HoverLostEvent e)
             UpdateState();
         }
 
+        private void onValueChanged(ValueChangedEvent<T> _) => UpdateState();
+        private void onDefaultChanged(ValueChangedEvent<T> _) => UpdateState();
+        private void onDisabledChanged(bool _) => UpdateState();
```

Comment:
Is there much of a reason why these are now three methods instead of inline lambdas like previously? I'm not sure the `-=` argument applies here if `UnbindAll` is used (it just sets the event delegates to `null`).

Reply:
Ah yeah leftover from previous method of doing that.